src/controllers/PhotoController.ts

```diff
@@ -16,6 +16,20 @@ class PhotoController implements IController {
     const photo = await this.photoService.create(photoPayload);
     res.status(201).json({ data: photo });
   };
+
+  public update: IRequestHandler = async (req, res) => {
+    const photoId = Number(req.params.id);
+    const photoPayload = req.body;
+    try {
+      const photoUpdated = await this.photoService.update(photoId, photoPayload);
+      res.status(200).json({
+        data:photoUpdated,
+      });
+    } catch (error) {
+      console.error(error);
```

Comment:
We should use Log class from utils, that way we can handle logging mechanisms from a central place, otherwise, if we ever want to modify this, we'd need to go and search for each `console.error` call.